src/components/item/index.tsx

```diff
@@ -88,7 +89,11 @@ const Product = ({
               />
             </Anchor>
           ) : (
-            <img src={imageString} alt="Leda MarketPlace." />
+            <img
+              src={imageString}
+              alt="Leda MarketPlace."
+              style={{ maxWidth: '400px', maxHeight: '800px' }}
```

Comment:
I"m ok with inline style. However, if this file already has a css file, use it. Let's keep the components as clean as possible